argo_prod/solrconfig.xml

```diff
@@ -81,7 +82,6 @@
 
         collection_title_tesim^5
 
-        objectId_tesim^5
```

Comment:
there is no point in boosting any sort of phrase or n-gram matches in a single token string.